kinbot/symmetry.py

```diff
@@ -140,19 +140,20 @@ def calculate_symmetry(species):
             cyc_atomid = [species.atomid[ci] for ci in cyc]
             symm = 0
             for i in range(len(cyc)):
-                new_order = np.roll(np.array(cyc_atomid),-i)
+                new_order = np.roll(np.array(cyc_atomid), -i)
                 if all([cyc_atomid[at] == new_order[at] for at in range(len(cyc))]):
                     symm += 1
-                new_order_reversed = np.roll(np.array(cyc_atomid[::-1]),-i)
+                new_order_reversed = np.roll(np.array(cyc_atomid[::-1]), -i)
                 if all([cyc_atomid[at] == new_order_reversed[at] for at in range(len(cyc))]):
                     symm += 1
             #additional patch: if an atom has two identical neighbors 
             #along the ring, but two distinct neighbors outside the ring
             #the symmetry number needs to be divided by 2
             if symm > 1:
                 divide = 1
+                print('cyclebug', species.chemid)
```

Comment:
Do we still need this print line?

Reply:
thanks, I removed it now, will push the corrected version soon.